features/supplier/view_and_edit_g_cloud_services.feature

```diff
@@ -0,0 +1,65 @@
+  And I click 'Save and return to service'
+  Then I am on the 'Changed cloud support service' page
+  And I see a success banner message containing 'You’ve edited your service. The changes are now live on the Digital Marketplace.'
+
+Scenario: Supplier user can edit the description of a service
+  Given I see the 'About your service' summary table filled with:
+    | field                        | value                                             |
+    | Service description          | Deliver digital transformation by the bucketload! |
+  When I click the top-level summary table Edit link for the section 'About your service'
+  Then I am on the 'About your service' page
+  And I enter 'This is an updated description' in the 'serviceDescription' field
+  And I click 'Save and return to service'
+  Then I am on the 'Test cloud support service' page
+  And I see a success banner message containing 'You’ve edited your service. The changes are now live on the Digital Marketplace.'
+  And I see the 'About your service' summary table filled with:
+    | field                        | value                          |
+    | Service description          | This is an updated description |
+
+Scenario: Supplier user can edit the features and benefits of a service
+  Given I see the 'Service features and benefits' summary table filled with:
+    | field                         | value                                                                                  |
+    | Service features and benefits | Service features Feature 1 Service benefits Benefit 1 Benefit 2  |
+  When I click the top-level summary table Edit link for the section 'Service features and benefits'
+  Then I am on the 'Service features and benefits' page
+  And I enter 'New Feature 2' in the 'input-serviceFeatures-2' field
+  And I enter 'Updated Benefit 2' in the 'input-serviceBenefits-2' field
+  And I click 'Save and return to service'
+  Then I am on the 'Test cloud support service' page
+  And I see a success banner message containing 'You’ve edited your service. The changes are now live on the Digital Marketplace.'
+  And I see the 'Service features and benefits' summary table filled with:
+    | field                         | value                                                                                  |
+    | Service features and benefits | Service features Feature 1 New Feature 2 Service benefits Benefit 1 Updated Benefit 2  |
+
+Scenario: Supplier can remove their G-Cloud service from the marketplace
+  When I click 'Remove service'
+  Then I see a destructive banner message containing 'Are you sure you want to remove your service?'
+
+  When I click 'Remove service'
+  Then I see a success banner message containing 'Test cloud support service has been removed.'
+
+  When I click 'Test cloud support service'
+  Then I see a temporary-message banner message containing 'This service was removed'
```

Comment:
Nice. Will this ensure that our setup step in the Background will only find one service?

Reply:
Yeah only one service is ever added for the supplier.